sos/report/plugins/ceph_mon.py

```diff
@@ -161,7 +151,7 @@ def get_ceph_version(self):
         )
         return 0
 
-    def get_ceph_ids(self):
+    def get_ceph_ids(self):  # To check
```

Comment:
Is this the last part to review before moving this PR out of draft?

Anything we can do to help with this, so we can make sure these changes are included in sos-4.5 which closes in about 1.5 weeks?

Reply:
Bumping this again. We'd like to get this into sos-4.5. Is there anything left to check here, @swalladge?

Reply:
Hi @TurboTurtle, my apologies I was distracted by other stuff and wasn't able to follow this up. We will double check what we need for the short term to get it into 4.5 and will update here.